internal/kafka/internal/metrics/component_versions_test.go

```diff
@@ -44,18 +45,63 @@ func TestVersionsMetrics_Collect(t *testing.T) {
 				},
 			}},
 			args: args{ch: make(chan prometheus.Metric, 100)},
-			want: 8,
+			want: 9,
 		},
 	}
 
 	for _, tt := range tests {
+		RegisterTestingT(t)
```

Comment:
please move this line above `for _, tt := range tests {`

Reply:
Got it, thanks @pawelpaszki